modules/retail/pages/retail-install-unified.adoc

```diff
@@ -60,13 +67,29 @@ In case of trouble, you might need to adjust the boot order in the BIOS.
 Review the settings and click btn:[Install].
 
 
-
-When the installation procedure has finished, you can check that all the required modules are added.
-Use the [command]``SUSEConnect --status-text`` command at a command prompt.
-For {smr}, the expected modules are:
-
-* {sles} Basesystem Module
-* Python 2 Module
-* Server Applications Module
-* SUSE Manager Proxy Module
-* SUSE Manager Retail Branch Server Module
+.Procedure: Configuration and registering of the Branch Server
+. Create an activation key based on the [systemitem]``SLE-Product-SUSE-Manager-Retail-Branch-Server-4.0-Pool`` base channel.
+For more information about activation keys, see xref:client-configuration:clients-and-activation-keys.adoc[Creating Activation Key].
+. From the [guimenu]``Child Channels`` listing select the recommended
```

Comment:
```suggestion
. In the [guimenu]``Child Channels`` listing, select the recommended channels by clicking the ``include recommended`` icon:
```

Reply:
I'll change it in my changes, but this is copy paste (with modified channel list) from https://github.com/SUSE/doc-susemanager/blob/master/modules/installation/pages/proxy-registration.adoc#suse-manager-proxy-registration

Reply:
Yeah, even tech writers write badly sometimes ;)